python/ql/test/experimental/query-tests/Security-old-dataflow/CWE-502/options

```diff
@@ -0,0 +1 @@
+semmle-extractor-options: --max-import-depth=2 -p ../lib
```

Comment:
I think the tests will fail due to `../lib` not being resolved in the same way as it used to. 

For our purposes, I would be happy to accept not having any tests of the old queries. It might be controversial, but that is how I feel about it right now :smile:

Reply:
Hah, I was anticipating the opposite, hence the inclusion of the tests.

Anyway, at least this is better than my first attempt, which only included the tests! :facepalm: